tools/scoring/score/dimensions/entity_point_identification.py

```diff
@@ -0,0 +1,159 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Core component """
+
+from score.dimensions.dimension import Dimension
+from score.types_ import DeserializedFile, EntityInstance, CloudDeviceId, PointsVirtualList
+from score.constants import FileTypes
+
+from typing import Set
+
+PROPOSED, SOLUTION = FileTypes
+
+
+class EntityPointIdentification(Dimension):
+  """
+  Quantifies whether the proposed file
+  included the correct points in each entity.
+  """
+  def _isolate_entities_virtual(self,
+                                file: DeserializedFile) -> Set[EntityInstance]:
+    return set(
+        filter(self.is_entity_canonical,
+               filter(self.is_entity_virtual, file.values())))
+
+  def _fetch_points_virtual(
+      self, file: DeserializedFile,
+      entities_virtual: Set[EntityInstance]) -> PointsVirtualList:
+    # For each canonically typed virtual entity
```

Comment:
you can use a docstring syntax instead of `#` for comments.

Reply:
Pylint flags these with `pointless-string-statement`. Should I use the `pylintrc` in the repo root or the one you linked above?